remo/reports/forms.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, *args, **kwargs):
         self.fields['campaign'].empty_label = 'Please select an initiative.'
         area_choices = FunctionalArea.active_objects.values_list('id', 'name')
         self.fields['functional_areas'].choices = area_choices
+        self.timezone = "America/New_York"
```

Comment:
Is it possible to use UTC (not London!) as default? That would match with the server time from what I know. Not really a TZ though.